src/wallet/wallet.h

```diff
@@ -137,9 +137,12 @@ enum WalletFlags : uint64_t {
 
     // will enforce the rule that the wallet can't contain any private keys (only watch-only/pubkeys)
     WALLET_FLAG_DISABLE_PRIVATE_KEYS = (1ULL << 32),
+
+    // Will enforce that the wallet should be blank
+    WALLET_FLAG_BLANK_WALLET = (1ULL << 33),
```

Comment:
Should we mention in the release notes that blank wallets can not be used in older bitcoin core versions?